Dockerfile

```diff
@@ -7,14 +7,16 @@ COPY ./src/Planthor.IdentityServerAspNetIdentity/*.csproj ./src/Planthor.Identit
 RUN dotnet restore
 
 # copy everything else and run release app
-COPY ./src/Planthor.IdentityServerAspNetIdentity/. /app/Planthor.IdentityServerAspNetIdentity/
-WORKDIR /app/Planthor.IdentityServerAspNetIdentity
+COPY ./src/Planthor.IdentityServerAspNetIdentity/. /app/src/Planthor.IdentityServerAspNetIdentity/
+WORKDIR /app/src/Planthor.IdentityServerAspNetIdentity
```

Comment:
Should we add a path src to planthor, the future we will add another moulde to this container?

Reply:
yes, we will have more project here (extensions + test)

Reply:
hmm, I am not 100% sure that we need extensions + test submodule in the release stage... 🤔. But okay with me